macro_utils_h_generator/macro_utils.tt

```diff
@@ -23,8 +23,13 @@
 #ifndef MACRO_UTILS_H
 #define MACRO_UTILS_H
 
+#ifdef __cplusplus
```

Comment:
>__cplusplus [](start = 7, length = 11)

consider then adding #ifdef __cplusplus / extern "C" { #endif to the potential function declarations/definitions that can arise form this file.

Reply:
Added

---
In reply to: [280658747](https://github.com/Azure/azure-macro-utils-c/pull/13#discussion_r280658747) [](ancestors = 280658747)